community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala

```diff
@@ -143,6 +143,7 @@ class CommunityBuildTestC extends CommunityBuildTest:
   @Test def geny = projects.geny.run()
   @Test def intent = projects.intent.run()
   @Test def minitest = projects.minitest.run()
+  @Test def monocle = projects.monocle.run()
```

Comment:
This should be moved to `CommunityBuildTestB` to be co-located with the dependencies of Monocle: (`cats`, `munit`, etc.), otherwise those dependencies need to be re-built from source in `CommunityBuildTestC` as well, creating duplicate work.